sdk/servicebus/azure-servicebus/stress/templates/network_loss.yaml

```diff
@@ -1,31 +1,26 @@
+{{- include "stress-test-addons.chaos-wrapper.tpl" (list . "stress.python-sb-network") -}}
+{{- define "stress.python-sb-network" -}}
 apiVersion: chaos-mesh.org/v1alpha1
-kind: NetworkChaos
-metadata:
-  name: '{{ .Release.Name }}-{{ .Release.Revision }}'
-  namespace: {{ .Release.Namespace }}
-  annotations:
-    experiment.chaos-mesh.org/pause: 'true'
-  labels:
-    scenario: 'stress'
-    release: '{{ .Release.Name }}'
-    revision: '{{ .Release.Revision }}'
+kind: Schedule
 spec:
-  scheduler:
-    cron: '@every 30s'
-  duration: '10s'
-  action: loss
-  direction: to
-  externalTargets:
-    - 'servicebus.windows.net'
-  mode: one
-  selector:
-    labelSelectors:
-      testInstance: "servicebus-{{ .Release.Name }}-{{ .Release.Revision }}"
-      chaos: 'true'
-    namespaces:
-      - {{ .Release.Namespace }}
-    podPhaseSelectors:
-      - 'Running'
-  loss:
-    loss: '100'
-    correlation: '100'
+  schedule: '@every 30s'
+  type: 'NetworkChaos'
+  networkChaos:
+    action: loss
+    mode: one
+    selector:
+      labelSelectors:
+        testInstance: "servicebus-{{ .Release.Name }}-{{ .Release.Revision }}"
+        chaos: 'true'
+      namespaces:
+        - {{ .Release.Namespace }}
+      podPhaseSelectors:
+        - 'Running'
+    duration: '10s'
+    direction: to
+    externalTargets:
+      - 'servicebus.windows.net'
```

Comment:
As we found, this doesn't work. Maybe in this update or a subsequent one, we can update this and the ARM template to use the same `{{ .Stress.ResourceGroupName }}` value?

Reply:
CC: @swathipil